src/api/routes/comments.ts

```diff
@@ -0,0 +1,52 @@
+import express, { Application, Request, Response, NextFunction } from 'express'
+const router = express.Router()
+//importing file reader fs
+const fs = require('fs')
+
+//endpoint for /GET requests for comments
+router.get('/', (req: Request, res: Response, next: NextFunction) => {
+//reading the contents of comments json file
+fs.readFile('./src/api/routes/comment.json', (err: string, data: string) => {
+    if (err) {
+      console.log(err)
+      res.status(500).json({
+          error: err
+      })
```

Comment:
Instead of simply calling `res.status(...).json(...)`, make sure to return it as well to make sure `res.json` isn't called twice i. e. The function terminates if an error occurs.
```js
return res.status(500).json({
    error: err
})
```

Reply:
I have added the return statement. Please inform if that was what you were asking.

Reply:
res.status() is now being returned in the function